### PR TITLE
Add etcd registry examples

### PR DESCRIPTION
**Pull request by georgehao**
1. Add etcd registry examples
2. add the way of Goland debug 
3. update README

### CODE REVIEW

File: golang/registry/etcd/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  module : "dubbogo user-info server"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "dev"
+
+registries :
+  "etcd":
+    protocol: "etcdv3"
+    timeout	: "3s"
+    address: "127.0.0.1:2380"
```
Comment (sxllwx) [inline golang/registry/etcd/README.md]: etcd http-api default listen in 2379.

File: golang/registry/etcd/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  module : "dubbogo user-info client"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "dev"
+
+registries :
+  "etcd":
+    protocol: "etcdv3"
+    timeout	: "3s"
+    address: "127.0.0.1:2380"
```
Comment (sxllwx) [inline golang/registry/etcd/README.md]: etcd http-api default listen in 2379.

Comment (sxllwx) [inline golang/registry/etcd/README.md]: etcd http-api default listen in 2379.

File: golang/registry/etcd/go-client/profiles/dev/client.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  module : "dubbogo user-info client"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "dev"
+
+registries :
+  "etcd":
+    protocol: "etcdv3"
+    timeout	: "3s"
+    address: "127.0.0.1:2380"
```
Comment (sxllwx) [inline golang/registry/etcd/go-client/profiles/dev/client.yml]: etcd http-api default listen in 2379

File: golang/registry/etcd/go-server/profiles/dev/server.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  module : "dubbogo user-info server"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "dev"
+
+registries :
+  "etcd":
+    protocol: "etcdv3"
+    timeout	: "3s"
+    address: "127.0.0.1:2380"
```
Comment (sxllwx) [inline golang/registry/etcd/go-server/profiles/dev/server.yml]: ~

**Review by AlexStocks — APPROVED** [review]

**Comment by sxllwx** [comment]
LGTM

**Review by sxllwx — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by fangyincheng — APPROVED** [review]
LGTM

**Pull request merged**